.github/workflows/build.yml

```
@@ -0,0 +1,157 @@
+        run: |
+          docker push registry.digitalocean.com/mindjunkies/django-mindjunkies-web --all-tags
+
+      - name: K8s cluster kubeconfig file with/ short-lived creds
+        run: |
+          doctl kubernetes cluster kubeconfig save --expiry-seconds 600 django-mindjunkies
+
+      - name: Update deployment secrets
+        run: |
+          echo "Creating .env.prod file from GitHub Secrets..."
+          cat << EOF > .env.prod
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)